src/go/cluster-to-redpanda-migration/cmd/root.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	vectorizedv1alpha1 "github.com/redpanda-data/redpanda/src/go/k8s/apis/vectorized/v1alpha1"
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"sigs.k8s.io/yaml"
+
+	"github.com/redpanda-data/redpanda/src/go/cluster-to-redpanda-migration/cmd/version"
+	"github.com/redpanda-data/redpanda/src/go/cluster-to-redpanda-migration/pkg/migration"
+)
+
+var (
+	defaultName            = "redpanda-cluster"
+	defaultNamespace       = "default"
+	DefaultChartRefVersion = "4.0.54"
+)
+
+type RootFlags struct {
+	ClusterFile string
+	ConsoleFile string
+	OutputFile  string
+
+	ChartVersion              string
+	RedpandaImageTag          string
+	RedpandaContainerRegistry string
+}
+
+func Execute() error {
+	flags := &RootFlags{}
+
+	// RootCmd represents the base command when called without any subcommands
+	root := &cobra.Command{
+		Use:   "cluster-to-redpanda-migration",
```

Comment:
nit: `k8s-cluster-crds-to-redpanda-migration` 

i'd suggest prefix k8s since `rpk` is a global cli for everything from cloud to self hosted ... maybe this leaves in a subspace  `rpk k8s  cluster-to-redpanda-migration` ? that would make sense as well. WDYT? @twmb thoughts?

Reply:
My understanding of this PR is that it's specifically for v1 to v2 operator migration -- we don't have `rpk k8s` yet at all, so worrying about `rpk k8s cluster-to-redpanda-migration` is a bit premature, because we'd only really support the v2 operator in rpk anyway?

Also as an aside -- cluster-to-redpanda doesn't imply anything about v1 to v2 operator, and I'm not sure why it's cluster-to-redpanda when both v1 and v2 and Redpanda clusters.

Reply:
cluster resource (v1 defined crd) to redpanda resource (v2). This is going to be thrown away after some time, i do not think we need to worry about long term maintenance or changes.